lib/vehicle_factory.rb

```diff
@@ -0,0 +1,16 @@
+class VehicleFactory
+  def initialize
```

Comment:
You do not need this method. When `VehicleFactory` is instantiated it has no attribute/args.